Add 2023-10-18 agenda and minutes (#586)
solid/specification: meetings/2023-10-18.md

* SC: we need to take Solid QA seriously, it is at the core of what we are doing. We need to prove that implementations are conforming with the specification. This is not a requirement for CGs. This work was for the WG to be able to advance on this, and prevent objections. Please let's create more energy around that.

### Joint Solid CG and Social CG meeting
URL: https://lists.w3.org/Archives/Public/public-solid/2023Oct/0036.html

* SC: Thread also at https://lists.w3.org/Archives/Public/public-swicg/2023Oct/0060.html
* SC: There is interest from participants in both groups. Let's make it so...
* SC: for example, the Linked Data Notification developed by the Social Web WG was done with collaboration from MIT at the time, and is compatible (and used) by both Solid and the Social Web people.
* SC: the work of each group is not conflicting with the other. E.g. Solid Protocol is about client-server interaction, while ActivityPub focuseS on server-server federation. A given server could possibly implement both.
* SC: SocialCG's meetings are on Friday. Our Tuesday (Special Topic Meeting slot) may be best.

> * SC: Any thoughts, suggestions, concerns?

Reviewer: ```suggestion
* SC: Thread also at https://lists.w3.org/Archives/Public/public-swicg/2023Oct/0060.html
  * ... There is interest from participants in both groups. Let's make it so...
  * ... for example, the Linked Data Notification developed by the Social Web WG was done with collaboration from MIT at the time, and is compatible with (and used by) both Solid and the Social Web people.
  * ... the work of each group is not conflicting with the other. E.g., Solid Protocol is about client-server interaction, while ActivityPub focuses on server-server federation. A given server could possibly implement both.
  * ... SocialCG's meetings are on Friday. Our Tuesday (Special Topic Meeting slot) may be best.
  * ... Any thoughts, suggestions, concerns?
```

> * eP: it might be useful to have a short pad for drafting topics, so that both side can prepare the meeting. Would make the meeting more productive. Example: authentication/authorization, media-types. About the server-server interaction: one of the server sometimes just acts as a client. All this could be prepared in advance.
> * SC: we did this for WebAgents. I agree we can do that. Authentication is a tricky topic: there are at least 4 ways how it is done (Fediverse, IndieAuth, Solid, ActivityPub). 

Reviewer: ```suggestion
* eP: It might be useful to have a short pad for drafting topics, so that both sides can prepare for the meeting. Would make the meeting more productive. Example: authentication/authorization, media-types. About the server-server interaction: one of the server sometimes just acts as a client. All this could be prepared in advance.
* SC: We did this for WebAgents. I agree we can do that. Authentication is a tricky topic; there are at least 4 ways it is done (Fediverse, IndieAuth, Solid, ActivityPub). 
```

* GH: It is so much wider. IETF people working on HTTP would also be interesting discussing with. I'm not saying we invite them all together, or a conference-style event, but may groups might be interested in joining this.
* SC: HTTP-Signature is also used in Mastodon and Solid. We need to look at the details.

> * eP: Solid is really focused on REST, while ActivityPub is more event oriented. In the Fediverse, many people use a client to interact with their own server only. In Solid, a given client may interact with many different servers. We should discuss those different approaches and the ideas behind them.
> * Hadrian: there are differences, but I don't see them as "fundamental".
> * SC: whether we discuss these topics or not with them, it is good to discuss them here. It was quite a difficult topic in the Social Web WG -- we won't solve it in one meeting!
> * SC: I'll see if our Tuesday meeting can work for them, as it is a 2h slot. I'll follow up on the mailing list.

Reviewer: ```suggestion
* eP: Solid is really focused on REST, while ActivityPub is more event-oriented. In the Fediverse, many people use a client to interact only with their own server. In Solid, a given client may interact with many different servers. We should discuss those different approaches and the ideas behind them.
* Hadrian: There are differences, but I don't see them as "fundamental".
* SC: Whether we discuss these topics with them or not, it is good to discuss them here. It was quite a difficult topic in the Social Web WG — we won't solve it in one meeting!
  * ... I'll see if our Tuesday meeting can work for them, as it is a 2h slot. I'll follow up on the mailing list.
```

* SC: https://github.com/solid/authorization-panel/pull/331
* SC: https://github.com/solid/webid-profile/pull/108

> * SC: https://github.com/solid/type-indexes/pull/28

Reviewer: ```suggestion
* SC: https://github.com/solid/authorization-panel/pull/331
  * ... https://github.com/solid/webid-profile/pull/108
  * ... https://github.com/solid/type-indexes/pull/28
```

* SC: There are a few other documents that I didn't get to checking. Please review or Any objections or merge?

> * SC: This is also mentioned in https://github.com/solid/specification/blob/main/meetings/2023-10-11.md#wg-charter-proposal-status-update . See related https://github.com/solid/solid-wg-charter/pull/58 .

Reviewer: ```suggestion
  * ... There are a few other documents that I didn't get to checking. Please review for any objections to merge.
  * ... This is also mentioned in [WG Charter Proposal Status Update](https://github.com/solid/specification/blob/main/meetings/2023-10-11.md#wg-charter-proposal-status-update). See related <https://github.com/solid/solid-wg-charter/pull/58>.
```

* SC: I will leave them for a while. The CG does not use the Solid process any more. The focus of the Solid Process on the Director raised some issues with the AC review of the WG charter proposal.

> * eP: I'm just noticing that the WG charter proposal points to solidproject.org/TR/ . This needs to be updated as well.

Reviewer: ```suggestion
* eP: I'm just noticing that the WG charter proposal points to [solidproject.org/TR/](https://solidproject.org/TR/). This needs to be updated as well.
```

> * SC: There is also an issue of using CG templates. WG charter could be updated to refer to EDs alternatively.
> * PAC: As I wrote in 58 "goes without saying" - the WG will follow the W3C Process and not other process. I'm not concerned about what those pointed documents say. They are just some input. The WG will create a new draft under the W3C Process based on that content. 

Reviewer: ```suggestion
* SC: There is also an issue of using CG templates. WG charter could be updated to refer to EDs.
* PAC: As I wrote in [pull#58](https://github.com/solid/solid-wg-charter/pull/58), "goes without saying" - the WG will follow the W3C Process and no other process. I'm not concerned about what those pointed documents say. They are just some input. The WG will create a new draft under the W3C Process based on that content. 
```

> * SC: Is it sufficient if we are updating the EDs rather than the ones in TRs?
> * PAC: There will be some text accompanying. In this text I can emphasis that the CG is using community-governance per CG charter. This move is occurring even if it is in the EDs, it is enough evidence for me.
> * SC: any objection to merging these PRs? All they do is remove the mention to the solid process?
> * eP: this is editorial change, so we can change only the third (patch) part of the version. Also, "editor's draft" is only for Working/Interest Groups. We should be using the CG templates.
> * SC: I've created an issue for that somewhere in w3c. They were supposed to update the process.
> * PAC: CG template mentions ED, e.g., https://www.w3.org/2021/12/rdf-star.html . Even if only mentioned for WGs, it wasn't the intention reserve that term.

Reviewer: ```suggestion
* SC: Is it sufficient if we update the EDs, rather than the ones in TR?
* PAC: There will be some accompanying text. In this text, I can emphasis that the CG is using community-governance, per CG charter. This move is occurring even if it is in the EDs; it is enough evidence for me.
* SC: Any objection to merging these PRs? All they do is remove the mention of the Solid process?
* eP: This is editorial change, so we can change only the third (patch) part of the version. Also, "editor's draft" is only for Working/Interest Groups. We should be using the CG templates.
* SC: I've created an issue for that somewhere in W3C. They were supposed to update the process.
* PAC: CG template mentions ED, e.g., [RDF-star and SPARQL-star: Final Community Group Report](https://www.w3.org/2021/12/rdf-star.html). Even if only mentioned for WGs, it wasn't the intention to reserve that term.
```

### WG Charter PRs from Reviews
* SC: We are over time, but the following PRs on the WG charter are mostly editorial. I would like to get rid of them. I'll assign them to PAC, and he can merge them if there are positive reviews after a week since their creation.
* SC: https://github.com/w3c/charter-drafts/issues/461 relates to https://github.com/w3c/charter-drafts/issues/459

> * PAC: thanks for bringing that up to the more general level. However, regardless of whether this is integrated in the template or not, I would prefer to keep it in our charter. Every charter discussion is different and justifies adjustments to the template. As for "It goes without saying", it could be counter-productive, but I want to emphasize the fact that we do not consider this as a "gift" that we are doing -- we know that this is how things are supposed to be.

Reviewer: ```suggestion
* PAC: Thanks for bringing that up to the more general level. Regardless of whether or not this is integrated in the template, I would prefer to keep it in our charter. Every charter discussion is different and justifies adjustments to the template. As for, "It goes without saying", it could be counter-productive, but I want to emphasize the fact that we do not consider this as a "gift" that we are doing — we know that this is how things are supposed to be.
```

#### extend timeline
URL: https://github.com/solid/solid-wg-charter/pull/57

#### clarify position wrt to other processes
URL: https://github.com/solid/solid-wg-charter/pull/58

#### add liaison with IETF HTTP WG
URL: https://github.com/solid/solid-wg-charter/pull/59

#### update link and charter text
URL: https://github.com/solid/solid-wg-charter/pull/60

### Solid CG Chair Election Procedure
URL: https://github.com/solid/specification/discussions/582

> * HZ: as a volunteer, I took a snapshot of the participants list, with a link to the Wayback machine.
> * SC: the list that you have is not accurate, it includes anybody. The procedure requires us to have only one participant from each affiliation. That's the part that I am trying to iron out with W3C staff at the moment.
> * HZ: then the procedure we agreed upon last week is not correct.
> * SC: the text says "only one participant per affiliation". Does the list you have comply with this?
> * HZ: I have a list of all participants, with affiliation. That's the only thing that W3C is providing. We can refine this list to make it compliant, but do we want every volunteer to do this independently?
> * SC: I would rather produce the list with W3C, and send it to each volunteer for them to check. We can go different ways: each organization chooses one representative voter (difficult); we pick one at random (not reproducible)...
> * eP: how many case do we have of multiple participants for one affiliation?
> * SC: I don't have the numbers, but we have several.
> * eP: suggest to reach out to the concerned organization.

Reviewer: ```suggestion
* HZ: As a volunteer, I took a snapshot of the participants list, with a link to the Wayback machine.
* SC: The list that you have is not accurate; it includes anybody. The procedure requires us to have only one participant from each affiliation. That's the part that I am trying to iron out with W3C staff at the moment.
* HZ: Then the procedure we agreed upon last week is not correct.
* SC: The text says "only one participant per affiliation". Does the list you have comply with this?
* HZ: I have a list of all participants, with affiliation. That's the only thing that W3C is providing. We can refine this list to make it compliant, but do we want every volunteer to do this independently?
* SC: I would rather produce the list with W3C, and send it to each volunteer for them to check. We can go different ways: each organization chooses one representative voter (difficult); we pick one at random (not reproducible)...
* eP: How many cases do we have of multiple participants for one affiliation?
* SC: I don't have the numbers, but we have several.
* eP: suggest to reach out to the concerned organization.
```
